source/index.html.md

```diff
@@ -969,9 +1052,8 @@ Examples:
 ### Merchant blocked by cardholder status
 
 A _merchant blocked by cardholder_ status code is both a decline of the current
-authorization, but also a notice to **halt all future
-merchant-initiated-transactions** on this card from this merchant. This
-includes, but is not limited to, recurring transactions.
+authorization, and also a notice to **halt all future merchant-initiated
+transactions in the series**.
```

Comment:
> **halt all future merchant-initiated transactions in the series**.

A stop request is even more powerful. 

 **halt all future merchant-initiated transactions from this merchant towards this cardholder**.